tests/pfcwd/test_pfcwd_function.py

```diff
@@ -551,3 +623,62 @@ def test_pfcwd_actions(self, request, fake_storm, setup_pfc_test, fanout_graph_f
                          PfcCmd.set_storm_status(self.dut, self.queue_oid, "disabled")
                      logger.info("--- Stop PFC WD ---")
                      self.dut.command("pfcwd stop")
+
+    def test_pfcwd_mmu_change(self, request, fake_storm, setup_pfc_test, fanout_graph_facts, ptfhost, duthosts, rand_one_dut_hostname, fanouthosts):
+        """
+        Tests if mmu changes impact Pfcwd functionality
+
+        Args:
+            request(object) : pytest request object
+            fake_storm(fixture) : Module scoped fixture for enable/disable fake storm
+            setup_pfc_test(fixture) : Module scoped autouse fixture for PFCwd
+            fanout_graph_facts(fixture) : fanout graph info
+            ptfhost(AnsibleHost) : ptf host instance
+            duthost(AnsibleHost) : DUT instance
+            rand_one_dut_hostname(string) : randomly pick a dut in multi DUT setup
+            fanouthosts(AnsibleHost): fanout instance
+        """
+        duthost = duthosts[rand_one_dut_hostname]
+        setup_info = setup_pfc_test
+        self.fanout_info = fanout_graph_facts
+        self.ptf = ptfhost
+        self.dut = duthost
+        self.fanout = fanouthosts
+        self.timers = setup_info['pfc_timers']
+        self.ports = setup_info['selected_test_ports']
+        key, value  = self.ports.items()[0]
+        self.ports = {key: value}
+        port = key
+        self.neighbors = setup_info['neighbors']
+        dut_facts = self.dut.facts
+        self.peer_dev_list = dict()
+        self.fake_storm = fake_storm
+        self.storm_hndle = None
+        logger.info("---- Testing on port {} ----".format(port))
+        self.setup_test_params(port, setup_info['vlan'], init=True, mmu_params=True)
+        self.stats = PfcPktCntrs(self.dut, "drop")
+
```

Comment:
pfcwd is started in storm_detect_path method. it will get started before the actual test starts in this case. Since I am reusing code, test_pfcwd_actions had a requirement to start watchdog later in one case